//////////////////////////////////////////////////////////////////////////////
// WARNING: This code is machine generated. Manual changes will be overridden.

syntax = "proto3";

package octaneapi;

option optimize_for = CODE_SIZE;

import "common.proto";
import "octaneenums.proto";

 // GRPC proto definitions for 'ApiTimeSampling' class from 'apitimesampling.h'
message ApiTimeSampling
{
    TimeArrayT pattern = 1;
    uint32 patternSize = 2;
    TimeT period = 3;
    AnimationType animationType = 4;
    TimeT endTime = 5;
    // Request packet corresponding to '[in] parameters in ApiTimeSampling::endTimeAuto'
    message endTimeAutoRequest
    {
    }

    // Response packet corresponding to '[out] parameters in 'ApiTimeSampling::endTimeAuto'
    message endTimeAutoResponse
    {
        
        float result = 1;
    }

}


// GRPC interface definition for class 'ApiTimeSampling' from 'apitimesampling.h'
service ApiTimeSamplingService
{
    rpc endTimeAuto(ApiTimeSampling.endTimeAutoRequest) returns (ApiTimeSampling.endTimeAutoResponse);
}
